outros-exercicios/livraria/src/test/java/com/example/livraria/model/AutorTest.java

```diff
@@ -0,0 +1,48 @@
+package com.example.livraria.model;
+
+import static com.example.livraria.model.enums.Sexo.MASCULINO;
+import static com.example.livraria.model.enums.Sexo.FEMININO;
+import static com.example.livraria.model.enums.Sexo.OUTROS;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class AutorTest {
+
+    Autor autor;
+    @DisplayName("Deve criar um autor válido")
+    @Test
+    void deveCriarAutorValido() {
+        autor = new Autor("Autor", OUTROS, 1990, 1);
+
+        assertEquals("Autor", autor.getNome());
+        assertEquals(OUTROS, autor.getSexo());
+        assertEquals(1990, autor.getAnoDeNascimento());
+        assertEquals(1, autor.getCpf());
```

Comment:
Ainda assim se for querer fazer teste na model, se colocar Equals e Hashcode na classe, poderá comparar o autor com apenas um assertEquals.
Também há assertEquals recursivo, caso não queira usar Equals e Hashcode.